integrations/destinations/mqtt.mdx

```diff
@@ -70,5 +70,6 @@ After the sink is created, you will continuously consume the data in the MQTT to
 | tls.client\_cert   | Optional. Path to the client's certificate file (PEM) or a string with the certificate content. Required for client authentication. Can use `fs://` prefix for file paths.                                                                              |
 | tls.client\_key    | Optional. Path to the client's private key file (PEM) or a string with the private key content. Required for client authentication. Can use `fs://` prefix for file paths.                                                                              |
 | topic              | Required. The topic name to subscribe or publish to. Can include wildcard topics, e.g., `/topic/#`.                                                                                                                                                     |
+| topic.field | Optional. Enables dynamic writing to multiple topics based on the value of a specified column. If set, the target topic is the value of this column.|
```

Comment:
IIUC, with `topic.field`, `topic` should also be optional.

Reply:
Checked with Xinhao, updated